src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -692,8 +692,9 @@ public class StripePaymentService : IPaymentService
             }).ToList();
         }
 
-        public async Task<string> AdjustStorageAsync(IStorableSubscriber storableSubscriber, int additionalStorage,
-            string storagePlanId)
+        private async Task<string> FinalizeSubscriptionChangeAsync(IStorableSubscriber storableSubscriber,
+            SubscriptionUpdateItemOptionsGenerator updateItemOptionsGenerator,
+            SubscriptionUpdateItemOptionsGenerator revertItemOptionsGenerator)
```

Comment:
Can't say I like generators in this context. But it's probably the one that fits best without doing major refactoring.

Reply:
changed patterns for you ;)